change/@fluentui-react-slider-63d1fa27-2328-4748-adc2-343d4e4f38c6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32424&commitId=e98798e89b68dbe92a0fa6bc2a9f23451b45b990&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.badgeMask.chromium.png|2| Changed|

</details>


  <details><summary><b>Field</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Slider.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>